### ISSUE TITLE
I couldn't install postcss-assets

### ISSUE THREAD

**Issue by ahoryakov** [borodean/postcss-assets#32]
Do:
```
npm install postcss-assets --save-dev
```
Result:
```
npm WARN addRemoteGit borodean/image-size resetting remote /Users/user/.npm/_git-remotes/git-github-com-borodean-image-size-git-12731099 because of error: xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
npm ERR! git clone --template=/Users/user/.npm/_git-remotes/_templates --mirror git://github.com/borodean/image-size.git /Users/user/.npm/_git-remotes/git-github-com-borodean-image-size-git-12731099: xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
npm WARN addRemoteGit borodean/image-size resetting remote /Users/user/.npm/_git-remotes/https-github-com-borodean-image-size-git-5f9ee4be because of error: xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
npm ERR! git clone --template=/Users/user/.npm/_git-remotes/_templates --mirror https://github.com/borodean/image-size.git /Users/user/.npm/_git-remotes/https-github-com-borodean-image-size-git-5f9ee4be: xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
npm WARN addRemoteGit borodean/image-size resetting remote /Users/user/.npm/_git-remotes/git-github-com-borodean-image-size-git-c4d823c8 because of error: xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
npm ERR! git clone --template=/Users/user/.npm/_git-remotes/_templates --mirror git@github.com:borodean/image-size.git /Users/user/.npm/_git-remotes/git-github-com-borodean-image-size-git-c4d823c8: xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
npm ERR! Darwin 15.0.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "postcss-assets" "--save-dev"
npm ERR! node v4.1.1
npm ERR! npm  v2.14.4
npm ERR! code 1

npm ERR! Command failed: git clone --template=/Users/user/.npm/_git-remotes/_templates --mirror git@github.com:borodean/image-size.git /Users/user/.npm/_git-remotes/git-github-com-borodean-image-size-git-c4d823c8
npm ERR! xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
npm ERR! 
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Applications/MAMP/htdocs/gulp-test/npm-debug.log
```

What i am doing wrong?

**Comment by borodean** [comment]
@ahoryakov as far as I see this issue is related to an image-size module fork.

Could you try [this thingie](https://github.com/borodean/postcss-assets/pull/30#issuecomment-144800339) and tell if it fixes the issue?

**Comment by ahoryakov** [comment]
Yes, this fix works.
```
"image-size": "https://github.com/borodean/image-size/archive/da2c863807a3e9602617bdd357b0de3ab4a064c1.tar.gz",
```

When the master version will work? :)

**Comment by borodean** [comment]
@ahoryakov finally, a new `image-size` repo was updated. I've released Assets 3.0.3 depending on it. Could you please try installing it verify it works for you?
